From b334a4b6be9ed98367c0c70821b5130247c4a5b9 Mon Sep 17 00:00:00 2001
From: Marko Lindqvist <cazfi74@gmail.com>
Date: Sun, 2 Nov 2025 10:43:09 +0200
Subject: [PATCH] Bump version number to 3.2.1b

Signed-off-by: Marko Lindqvist <cazfi74@gmail.com>
---
 fc_version | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/fc_version b/fc_version
index bae71ba06d..77a85a4ee4 100755
--- a/fc_version
+++ b/fc_version
@@ -9,19 +9,19 @@ MAJOR_VERSION="3"
 MINOR_VERSION="2"
 PATCH_VERSION="1"
 EMERGENCY_VERSION=""
-VERSION_LABEL=""
+VERSION_LABEL="b"
 
 # Type is either "development" or "stable"
 # Date and NEWS URL are relevant only for "stable" releases
-if test "${VERSION_LABEL}" = "" ; then
+#if test "${VERSION_LABEL}" = "" ; then
   RELEASE_TYPE="stable"
   RELEASE_DATE="2025-10-10"
   NEWS_URL="https://www.freeciv.org/wiki/NEWS-${MAJOR_VERSION}.${MINOR_VERSION}.${PATCH_VERSION}"
-else
-  RELEASE_TYPE="development"
+#else
+#  RELEASE_TYPE="development"
   #RELEASE_DATE="2025-07-10"
   #NEWS_URL="https://www.freeciv.org/wiki/NEWS-3.2.0"
-fi
+#fi
 
 #
 # At the time being these same updates need to be made to the meson.build,
-- 
2.51.0

